reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-74.md

```diff
@@ -0,0 +1,114 @@
+---
+title: What's New in PowerShell 7.4 (preview)
+description: New features and changes released in PowerShell 7.4 (preview)
+ms.date: 01/05/2023
+---
+
+# What's New in PowerShell 7.4 (preview)
+
+PowerShell 7.4-preview.1 includes the following features, updates, and breaking changes.
+
+For a complete list of changes, see the [Change Log][07] in the GitHub repository.
+
+## Breaking Changes and Improvements
```

Comment:
nit: Until we have some improvements, maybe best to say only changes here?

Reply:
Agreed